docs/crds/hostrule.md

```diff
@@ -122,6 +124,16 @@ prior to this CRD creation. The application profile should be of `TYPE` of `APPL
  This property can be applied only for secure FQDNs and cannot be applied for insecure routes.
  The application profiles can be used for various HTTP/HTTP2 protocol settings.
 
+#### Express custom ICAP profile
+
+HostRule CRD can be used to express an ICAP profile reference. The ICAP profile reference should have been created in the Avi Controller prior to this CRD creation.
+
+        icapProfile: 
```

Comment:
Better to mention that we will be considering only one icap profile reference from the list of references.

Reply:
I've changed it to mention explicitly that a single reference per host is allowed